backend/de.metas.acct.base/src/main/sql/postgresql/ddl/functions/m_inventoryline_update_qtybook_from_fact_acct.sql

```diff
@@ -204,7 +204,7 @@ BEGIN
                NULL                                                                                   AS externalid,
                NULL                                                                                   AS costprice,
                --
-               prev_invl.qtybook                                                                      AS qtybook,
+               0                                                                                      AS qtybook,
```

Comment:
are u considering QtyBook=0 because you will recompute it anyways?

Reply:
is it also because in this way we make sure we set QtyBook=0 for those lines which are not present in fact acct ?